website/content/sdk/guides/intros/crypto/discovery.md

```diff
@@ -0,0 +1,131 @@
+|  63 | zcash         | zec      | Zcash         | https://assets.coingecko.com/coins/images/486/large/circle-zcash-color.png?1547034197 |      46.24      |    602735976 |                64 |               9.69752e+08 |    3.57518e+07 |   47.32      |   45.31      |        0.479718    |                       1.04838 |             5.91658e+06 |                            0.99135 |          1.30523e+07 |        2.1e+07 |      2.1e+07 |  3191.93     |                -98.5521 | 2016-10-29T00:00:00.000Z | 19.75       |                133.959  | 2020-03-13T02:20:55.002Z |                                                                                   | 2022-12-05T22:57:30.607Z |                                  21.4551  |                                 0.502131 |                                 -75.4374 |                                   -53.9461 |                                  1.04838  |                                  -14.1037 |                                13.3262   |
+| 141 | yearn-finance | yfi      | yearn.finance | https://assets.coingecko.com/coins/images/11849/large/yfi-192x192.png?1598325330      |    7082.34      |    220754373 |               142 |               2.59099e+08 |    4.54938e+07 | 7472.6       | 7052.76      |     -157.22        |                      -2.17168 |            -5.96275e+06 |                           -2.63004 |      31239.8         |    36666       |  36666       | 90787        |                -92.1948 | 2021-05-12T00:29:37.713Z | 31.65       |              22292.3    | 2020-07-18T12:26:27.150Z |                                                                                   | 2022-12-05T22:57:24.815Z |                                  15.7094  |                                -0.124164 |                                 -71.239  |                                   -22.0297 |                                 -2.17168  |                                  -15.4178 |                                13.2592   |
+
+Coins can be filtered by category. For a list of defined categories enter:
+
+```python
+categories:dict = openbb.crypto.disc.categories_keys()
+```
+
+```python
+categories[10]
+```
+
+```console
+'big-data'
+```
+
+```python
+openbb.crypto.disc.coins(category = categories[10], sortby = 'market_cap').head(5)
```

Comment:
There something off here, this shouldn't all be different code blocks, I'd group them all together. E.g. I can not run these individually anyways.

Reply:
I removed the part showing the category name; now they are just two separate commands. Better?